runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -179,14 +224,14 @@ class LayoutFFITypeHelpers
 	}
 
 	/**
-	 * @brief Create a FFI type from a layout string object for both primitive and struct
+	 * @brief Create a FFI type from a layout string object for both primitive and struct.
 	 *
-	 * @param typeFFI[in] The custom FFI type to be created
+	 * @param typeFFI[out] The custom FFI type to be created
 	 * @param layoutStringObject[in] An object containing the layout string describing the field types.
 	 * @return The size of the layout, extracted from the layout string, UDATA_MAX if a failure occurs
 	 */
 	VMINLINE UDATA
-	getLayoutFFIType(ffi_type** typeFFI, j9object_t layoutStringObject)
+	getLayoutFFIType(ffi_type **typeFFI, j9object_t layoutStringObject)
```

Comment:
Please correct the type of the local on line 245:
```c
		const char *layoutTemp = layout;
```

Reply:
Fixed.